cpu/sam0_common/periph/spi.c

```diff
@@ -149,6 +152,13 @@ int spi_acquire(spi_t bus, spi_cs_t cs, spi_mode_t mode, spi_clk_t clk)
 
 void spi_release(spi_t bus)
 {
+    /* disable the device */
+    dev(bus)->CTRLA.reg &= ~(SERCOM_SPI_CTRLA_ENABLE);
+    while (dev(bus)->SYNCBUSY.reg & SERCOM_SPI_SYNCBUSY_ENABLE) {}
```

Comment:
Since we just shut off the SERCOM afterwards there should be no need to wait for `SYNCBUSY`.

Reply:
I'd like to keep this line here because we're not sure if calling acquire right after release will be reliable. Since some bits are enable protected in the CTRLA register, better to be 100% sure we disable it first.

Reply:
At least the SAMD5x/E5x manual states "When the SPI is being disabled, writing to these registers will be completed after the disabling.", so removing the sync might be safe after all. I left it in though.